components/vc/front/src/torque.c

```diff
@@ -126,6 +287,13 @@ torque_launchControl_E torque_getLaunchControlState(void)
     return torque_data.launch_control_state;
 }
 
+bool torque_isLaunching(void)
+{
+    const bool launching = (torque_data.launch_control_state != LAUNCHCONTROL_INACTIVE) &&
+                           (torque_data.launch_control_state != LAUNCHCONTROL_REJECTED);
```

Comment:
same here, be explicit about allowed states in case you add a state in the future and forget to add it here.